### DISCUSSION TITLE
Plugins Development

### DISCUSSION THREAD

**Discussion by wigsaparelli** [jgraph/drawio#3346] [Q&A]
Hi, I know the drawio repository is open source but closed to contributions but does that also apply to plugins?
I, and we at SquaredUp, would love to be able to author plugins to extended / customise functionality...

**Comment by davidjgraph** [comment]
Yes, it applies (especially) to plugins

**Reply by SomniVertix** [reply]
So does this mean that in order to develop plugins, we would need to fork? Or is there a standard way to develop plugins?

**Reply by davidjgraph** [reply]
No, you can develop a plugin and use it. We just don't accept PRs is what the original question is about.

**Reply by SomniVertix** [reply]
Ah okay - Are there any guides/documentation around developing a plugin? I'm not sure if I just can't find it

**Reply by SomniVertix** [reply]
@davidjgraph  looping back on this - Is there any hints that can help point me in the right direction?
